src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,86 @@
+import styled from "styled-components";
+import facebook from "@public/icons/facebook.png";
+import instagram from "@public/icons/instagram.png";
+import tiktok from "@public/icons/tiktok.png";
+import youtube from "@public/icons/youtube.png";
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+const Main = styled.div`
```

Comment:
Try separating this in a separate file. Import it as `import * as S from "./styles"`. Do it for every styled-component below

Reply:
Also, I now added a theme to our project. Try grabbing values such as fontsizes, colors, spacing and radius directly from the theme